modules/builtin/src/hooks/on_stage_request/.pipeline_approval.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+/**
+ * Gives a short demonstration of a pipeline approval process.
+ *
+ * @param bp The botpress SDK
+ * @param bot The complete configuration of the bot
+ * @param users The list of users of that workspace (email, role)
+ * @param pipeline The list of configured stages
+ * @param hookResult The result of the hook which contains actions
+ */
+const stageChangeRequest = async () => {
+  const request_user = users.find(u => u.email == bot.pipeline_status.stage_request.requested_by)
+
+  // By default, we want to keep the bot in the current stage
+  hookResult.actions = []
+
+  const stageRequest = bot.pipeline_status.stage_request
+  stageRequest.approvers = stageRequest.approvers || _getApprovers()
```

Comment:
That field should be added to the Workspace object typings

Reply:
Done in c6a8e414745315d4ef12e8fd4c82a6605e6ef2d7